### ISSUE TITLE
Olecranon fracture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10205]
**Preferred term label:**
Olecranon fracture

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0555335

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Osteogenesis imperfecta

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000002
- label: Olecranon fracture
- parents: HP:0041219
- pmid: PMID:30725980
- def: A partial or complete break in the olecranon,which is the proximal articular portion of the ulna and makes up part of the elbow.
- comment: The olecranon is the proximal articular portion of the ulna. Along with the coronoid process, it forms the greater sigmoid (semilunar) notch which articulates with the humerus to provide flexion and extension of the elbow.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10205
- synonyms:
